frontend/src/components/ValueUnitField.tsx

```diff
@@ -64,7 +64,7 @@ const ValueUnitField: React.FC<ValueUnitFieldProps> = ({
             })
           }
           onChange={(value) => {
-            onChange(`${value || minAsNumber}${currentUnitOption.unit}`);
+            onChange(`${value || minAsNumber || ''}${currentUnitOption.unit}`);
```

Comment:
Should be using `??` as `0` will be falsey.
```suggestion
            onChange(`${value ?? minAsNumber ?? ''}${currentUnitOption.unit}`);
```